From f9ad72e4d7b99072de1bd9af737371bca4f55bfe Mon Sep 17 00:00:00 2001
From: Yannick Fertre <yannick.fertre@foss.st.com>
Date: Tue, 24 Sep 2024 14:51:45 +0200
Subject: [PATCH] arm64: dts: st: fix support of continuous display on
 stm32mp257 EV1 board

Set by default on the lvds panel.

Change-Id: I7efa759a25e6d8393b00c512bb0d2715d8b74230
Signed-off-by: Yannick Fertre <yannick.fertre@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/406523
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
---
 arch/arm64/boot/dts/st/stm32mp257f-ev1.dts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts b/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
index f4032a1459fd..66f981326c4f 100644
--- a/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
+++ b/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
@@ -146,6 +146,7 @@ panel_lvds: panel-lvds {
 		compatible = "edt,etml0700z9ndha", "panel-lvds";
 		enable-gpios = <&gpiog 15 GPIO_ACTIVE_HIGH>;
 		backlight = <&panel_lvds_backlight>;
+		default-on;
 		status = "okay";
 
 		width-mm = <156>;
-- 
2.50.1

